### DISCUSSION TITLE
New TitleScreen.bik HD restored MOD.

### DISCUSSION THREAD

**Discussion by drtechxx** [emoose/OutRun2006Tweaks#240] [Show and tell]
### Describe the feature you'd like to see added

First of all, I apologize if I didn't post this in the right place.

OutRun 2006 Tweak looks really nice with the new HD textures. But what bothered me a lot was the presentation of the title screen, which was still in SD (640x480). So I decided to make a small improvement in the resolution and colors. It's much better than the original, but don't expect phenomenal quality; the original has very poor quality and unfortunately I couldn't go much further than that. I believe that using professional video editing programs, the quality can be even better.

Before you do, back up the TitleScreen.bik file from the nv folder.

Ps. I couldn't remove the white lines on the left and top edges that were present in the original version.

https://drive.google.com/file/d/1DOChtkt_uxpojIJmJgFlqPQheI_ypWcZ/view?usp=sharing

**Comment by Tenome** [comment]
Misc. modifications go in https://github.com/emoose/OutRun2006Tweaks/issues/20 (despite the name, it also has other mods in it).

**Comment by drtechxx** [comment]
Could the moderator move the thread to the correct location?

**Comment by Tenome** [comment]
Just repost that there and close this thread.

**Comment by Tenome** [comment]
Having trouble there? lol

**Comment by Zoukambada** [comment]
@Tenome I didn't check this mod yet but if anyone wants to try, I discovered that the PS2 USA-NTSC version has a titlescreen with a higher resolution than the vanilla PC one. It's located in the MV folder as well, but the file has to be extracted from there with a Hex Editor. I tried but couldn't get the music working, the video played without sound, so probably I messed something up or the PS2 uses a different audio codec. Maybe the way is to use the PC audio with the PS2 movie visuals and create a new proper .BIK.

**Comment by Tenome** [comment]
@Zoukambada Worth taking a look at I suppose, can you upload it? Sound is an easy fix, just mux it in from elsewhere.

**Comment by Zoukambada** [comment]
> @Zoukambada Worth taking a look at I suppose, can you upload it? Sound is an easy fix, just mux it in from elsewhere.
@Tenome here it is:

**The intro is from SLUS_212.74**
https://mega.nz/folder/mQxl3KKZ#Us3TKcicnSUxt6EnZO1N_Q

**Files in the folder:**
- MV.PS2 - The unmodified file, doesn't play on VLC.
- TitleScreen.PS2.bik - The modified file, it's the same one but at the beginning of the file (header?) I added the word BIK so it can be played on VLC.

**About the video:**
- The resolution isn't higher (It is still 640x480), but the image quality is better than the PC version. If you take a look in the LOGO it is crispier and not blurred as the PC one.
- VLC says it is 25 fps.
- The video has no sound in VLC.
- The same video plays twice.

IMHO I prefer not using the AI upscale because it deforms the fonts and the graphics turning them almost like an amateur painting. There must be other tools to upscale that will be more faithful to the source material.

I hope it helps you guys out.

**Comment by emoose** [comment]
We did look at the O2SP JP titlescreen before in the discord, iirc it seemed to have sharper logo screens but the gameplay sections were lower quality, not sure if the SLUS release uses the same files though

https://gofile.io/d/iUJXYj for the JP movies. (E: oh that's dead now, reup at https://gofile.io/d/Oqt0UV)

The Xbox movies also had sharper logos but they did some weird cropping in the gameplay sections, it could still be worth taking logos from one of the console versions though.
